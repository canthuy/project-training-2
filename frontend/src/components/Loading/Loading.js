import { Spinner } from 'react-bootstrap';
import {SpinnerSC} from './Loading.styles';

const Loading = () => {
  return (
    <SpinnerSC>
      <Spinner animation="border" />
    </SpinnerSC>
  );
};

export default Loading;
