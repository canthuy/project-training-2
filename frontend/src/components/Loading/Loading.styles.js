import styled from 'styled-components';

const SpinnerSC = styled.div`
  text-align: center;
  margin: 10rem auto;
`;

export { SpinnerSC };
