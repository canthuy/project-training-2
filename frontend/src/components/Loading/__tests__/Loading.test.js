import React from 'react';
import { render } from '@testing-library/react';
import Loading from '../Loading';
window.React = React;

describe('Loading component', () => {
  it('render component', () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
