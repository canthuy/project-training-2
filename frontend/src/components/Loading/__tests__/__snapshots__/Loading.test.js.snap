// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading component render component 1`] = `
<div>
  <div
    class="sc-bdvvtL heGnFz"
  >
    <div
      class="spinner-border"
    />
  </div>
</div>
`;
